Homework_7/Task-34.py

```diff
@@ -0,0 +1,32 @@
+# * Задача 34: *
+# Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм.
+# Поскольку разобраться в его кричалках не настолько просто,
+# насколько легко он их придумывает, Вам стоит написать программу.
+# Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв)
+# в каждой фразе стихотворения одинаковое.
+# Фраза может состоять из одного слова,
+# если во фразе несколько слов, то они разделяются дефисами.
+# Фразы отделяются друг от друга пробелами.
+# Стихотворение Винни-Пух вбивает в программу с клавиатуры.
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”,
+# если с ритмом все не в порядке.
+
+# *Пример:*
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
+# **Вывод:** Парам пам-пам  
+
+# Гласных букв в русском языке 10: а, я, у, ю, о, е, ё, э, и, ы.
+
+glasn = {'а', 'я', 'у', 'ю', 'о', 'е', 'ё', 'э', 'и', 'ы'}
```

Comment:
Можно было задать простой строкой
```suggestion
glasn = 'аяуюоеёэиы'
```

Reply:
Напоминаю, что транслитерация русских слов не используется в названии переменных.

Reply:
Принято, будем стараться.